pydantic_ai_slim/pydantic_ai/_cli.py

```diff
@@ -134,8 +134,8 @@ def cli(args_list: Sequence[str] | None = None) -> int:  # noqa: C901  # pragma:
     The current date and time is {datetime.now()} {tzname}.
     The user is running {sys.platform}.""",
         )
-    except UserError:
-        console.print(f'[red]Invalid model "{args.model}"[/red]')
+    except UserError as e:
+        console.print(f'[red]Invalid model[/red] [magenta]{args.model}[/magenta]:\n{e}')
```

Comment:
The output of this is a bit repetitive tho.

Reply:
without it you have no idea why it's failing.

Reply:
I mean, my point is more about the repetition, see the failing test. I'm not saying to not change it...